code/modules/mob/living/status_procs.dm

```diff
@@ -0,0 +1,512 @@
+	SetEarDamage(new_value)
+
+// EAR_DEAF
+
+/mob/living/EarDeaf(amount)
+	SetEarDeaf(max(ear_deaf, amount))
+
+/mob/living/SetEarDeaf(amount)
+	ear_deaf = max(amount, 0)
+
+/mob/living/AdjustEarDeaf(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(ear_deaf, amount, bound_lower, bound_upper)
+	SetEarDeaf(new_value)
+
+// EYE_BLIND
+
+/mob/living/EyeBlind(amount)
+	SetEyeBlind(max(eye_blind, amount))
+
+/mob/living/SetEyeBlind(amount)
+	var/old_val = eye_blind
+	eye_blind = max(amount, 0)
+	// We transitioned to/from 0, so update the eye blind overlays
+	if((old_val == 0 || eye_blind == 0) && (old_val != eye_blind))
+		update_blind_effects()
+
+/mob/living/AdjustEyeBlind(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(eye_blind, amount, bound_lower, bound_upper)
+	SetEyeBlind(new_value)
+
+// EYE_BLURRY
+
+/mob/living/EyeBlurry(amount)
+	SetEyeBlurry(max(eye_blurry, amount))
+
+/mob/living/SetEyeBlurry(amount)
+	var/old_val = eye_blurry
+	eye_blurry = max(amount, 0)
+	// We transitioned to/from 0, so update the eye blur overlays
+	if((old_val == 0 || eye_blurry == 0) && (old_val != eye_blurry))
+		update_blurry_effects()
+
+/mob/living/AdjustEyeBlurry(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(eye_blurry, amount, bound_lower, bound_upper)
+	SetEyeBlurry(new_value)
+
+// HALLUCINATION
+
+/mob/living/Hallucinate(amount)
+	SetHallucinate(max(hallucination, amount))
+
+/mob/living/SetHallucinate(amount)
+	hallucination = max(amount, 0)
+
+/mob/living/AdjustHallucinate(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(hallucination, amount, bound_lower, bound_upper)
+	SetHallucinate(new_value)
+
+// JITTER
+
+/mob/living/Jitter(amount, force = 0)
+	SetJitter(max(jitteriness, amount), force)
+
+/mob/living/SetJitter(amount, force = 0)
+	// Jitter is also associated with stun
+	if(status_flags & CANSTUN || force)
+		jitteriness = max(amount, 0)
+
+/mob/living/AdjustJitter(amount, bound_lower = 0, bound_upper = INFINITY, force = 0)
+	var/new_value = directional_bounded_sum(jitteriness, amount, bound_lower, bound_upper)
+	SetJitter(new_value, force)
+
+// LOSE_BREATH
+
+/mob/living/LoseBreath(amount)
+	SetLoseBreath(max(losebreath, amount))
+
+/mob/living/SetLoseBreath(amount)
+	losebreath = max(amount, 0)
+
+/mob/living/AdjustLoseBreath(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(losebreath, amount, bound_lower, bound_upper)
+	SetLoseBreath(new_value)
+
+// PARALYSE
+
+/mob/living/Paralyse(amount, updating = 1, force = 0)
+	SetParalysis(max(paralysis, amount), updating, force)
+
+/mob/living/SetParalysis(amount, updating = 1, force = 0)
+	if(status_flags & CANPARALYSE || force)
+		paralysis = max(amount, 0)
+		if(updating)
+			update_canmove()
+			update_stat()
```

Comment:
Paralysis knocks someone out. `update_canmove` does not handle consciousness/unconsciousness state transitions, only the raw ability to move. `update_stat` updates whether someone is unconscious or not.